python/pyspark/cloudpickle/cloudpickle.py

```diff
@@ -236,7 +236,7 @@ def _extract_code_globals(co):
         out_names = {names[oparg] for _, oparg in _walk_global_ops(co)}
 
         # Declaring a function inside another one using the "def ..."
-        # syntax generates a constant code object corresponding to the one
+        # syntax generates a constant code object corresonding to the one
```

Comment:
typo?

Reply:
Its a copy from cloudpickle. I think I would just keep it as is.

Reply:
https://github.com/cloudpipe/cloudpickle/pull/406. Let's fix it back when we upgrade next time.